metadoc-js/src/main/scala/metadoc/Metadoc.scala

```diff
@@ -68,11 +68,16 @@ object MetadocApp extends js.JSApp {
 
     val options = jsObject[IEditorConstructionOptions]
     options.readOnly = true
-    options.value = contents
-    options.language = "scala"
 
     val editor = monaco.editor.Editor.create(app, options)
 
+    val model = monaco.editor.Editor.createModel(
+      value = contents,
+      language = "scala",
+      uri = monaco.Uri.parse(s"file:$fileName")
```

Comment:
I agree, we can do that once #19 is done, I still haven't wrapped my head around IModel and content providers